builder.md

```diff
@@ -81,39 +86,64 @@ Each template repository MUST have a templates folder containing a config.yaml a
 These files have the following structure:
 
     default:
-        extension: .file-extension
-        output: output-directory-name
+      supported-systems: [base16]
+      filename: "output-directory-name/{{ scheme-system }}-{{ scheme-slug }}.file-extension"
 
     additional:
-        extension: .file-extension
-        output: output-directory-name
+      extension: .another-extension
+      output: output-directory-name
+
+This example specifies that a Builder is to parse two template files: `templates/default.mustache` and `templates/additional.mustache`.
+
+`filename` defines a mustache template which returns a filename relative to the template repository's root directory. All the [template variables](#template-variables) listed below are available.
```

Comment:
Is this the place to mention what happens when there are `filename`s with equivalent values?